apps/hash/README.md

```diff
@@ -86,13 +75,21 @@ To run HASH locally, please follow these steps:
 
 1. [Clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) this repository and **navigate to the root of the repository folder** in your terminal.
 
+1. We use [mise-en-place](https://mise.jdx.dev/) to manage tool versions consistently across our codebase. We recommend using `mise` to automatically install and manage the required development tools:
+
+   ```sh
+   mise install
+   ```
+
+   It's also possible to install them manually, use the correct versions for these tools as specified in `.config/mise`.
+
 1. Enable [corepack](https://nodejs.org/api/corepack.html):
 
-```sh
-corepack enable
-```
+   ```sh
+   corepack enable
+   ```
 
-You might need to re-create your shell.
+   You might need to re-create your shell.
```

Comment:
Do we want to add the mise env flag as well?

Reply:
No, because that would only apply when you install `node` via mise. For now we can keep it explicit and add the setting to mine when it's available.